### ISSUE TITLE
Squad building and http://xwing-builder.co.uk/ interaction

### ISSUE THREAD

**Issue by oszymanezyk** [tjakubo2/TTS_xwing#28]
Squad building and http://xwing-builder.co.uk/ interaction.

I built a list using the xwing-builder page, and i followed the instructions provided in the mod to spawn the list (  xwing-builder page -> "Forum or email" -> Plain text (brief) ). The list that I try to spawn is detailed below.  After clicking the "spawn it" button, no ships appear on the board. A new note appears with the following text: 

Source bags ready
Input parsed
Recognized uk_Browse format
Core spawned
Core validated
Accesories spawned
Accesories validated
Miscellanous spawned
[b]Success![/b]

I tried with different lists and different ships, and different output formats that the xwing builder page offers (from plain text to bb code, either copying the whole thing, or only parts of it.). None of them seem to work, and the issue can be reproduced with any lists. 

This would be an example list built on http://xwing-builder.co.uk/ using the plain text brief format, and cropped to only include pilots and upgrades:

Access to list:
http://xwing-builder.co.uk/view/693235/reypoebb8

Rey [Push the Limit, Kanan Jarrus, Finn, Millennium Falcon (Segnor's Loop Version), Engine Upgrade] (61)
Poe Dameron (PS9 Version) [Black One, BB-8, Push the Limit, Integrated Astromech] (39)

The example squad with Poe and Hello Asty given in the mod spawns perfectly. However, I cannot seem to get any lists spawned using any of the http://xwing-builder.co.uk/ outputs. 

HTH to reproduce the bug, otherwise I'm happy to help either in-game, or to give more details.

**Comment by tjakubo2** [comment]
Hey oszymanezyk, this is similiar to #27.
You have to copy the whole snippet, just like the picture from [Doc: Squad Building](https://github.com/tjakubo2/TTS_xwing/wiki/Doc:-Squad-Building) instructs: http://i.imgur.com/oNoJgLX.png

If you click the "Copy plain text to clipboard button" and proceed as usual, pasted snippet will spawn your ships :)
Please confirm if you get this to work, cheers!

**Comment by oszymanezyk** [comment]
Hey tjakubo2, I tried these and your instructions earlier following the squad builder instructions, and copy pasted everything. It didn't work, and it behaved the very same as in my first post:

I found a simple solution to this problem though:

I decided to clean up my "Documents/My Games/Tabletop Simulator" folder, removed any traces of the xwing mod, and any other deprecated x-wing related mods in Tabletop Simulator. I reinstalled the mod, and everything works now perfectly and smoothly.

Thanks for the quick help!
